lib/internal/child_process.js

```diff
@@ -912,7 +928,15 @@ function setupChannel(target, channel, serializationMode) {
   };
 
   function emit(event, message, handle) {
-    target.emit(event, message, handle);
+    const args = [event, message, handle];
```

Comment:
nit: we probably don't need to store variable now that we only use it once.

Reply:
fixed